cli/js/web/console.ts

```diff
@@ -96,19 +99,146 @@ function createIterableString<T>(
   }
   ctx.add(value);
 
-  const entries: string[] = [];
+  let entries: string[] = [];
   // In cases e.g. Uint8Array.prototype
   try {
     for (const el of value) {
       entries.push(config.entryHandler(el, ctx, level + 1, maxLevel));
     }
   } catch (e) {}
   ctx.delete(value);
+  if(entries.length > MAX_ARRAY_LENGTH){
+    const nmore = entries.length - MAX_ARRAY_LENGTH;
+    entries = entries.slice(0, MAX_ARRAY_LENGTH);
+    entries.push(`... ${nmore} more items`);
+  }
   const iPrefix = `${config.displayName ? config.displayName + " " : ""}`;
-  const iContent = entries.length === 0 ? "" : ` ${entries.join(", ")} `;
+  let iContent;
+  if(config.group){
+    iContent = groupEntries(entries, level, value);
+    const indetation = "  ".repeat(level + 1);
```

Comment:
indentation is only used if `iContent.legnth > 1` and so should only be declared in the next block